src/sentry/templates/sentry/layout.html

```diff
@@ -56,36 +55,6 @@
         {% show_system_status %}
     {% endif %}
 
-    {% block global_sidebar %}
-      {% if request.user.is_authenticated %}
-        <div id="blk_sidebar"></div>
-        <script>
-        $(function(){
-          var SidebarWrapper = createReactClass({
-            propTypes: {
-              organization: PropTypes.any.isRequired
-            },
-            getChildContext: function () {
-              return {
-                organization: this.props.organization,
-              }
-            },
-            render: function () {
-              return React.createElement(SentryApp.OrganizationsLoader, {},
```

Comment:
This might be the only usage of OrganizationsLoader left as well.

Reply:
Nice it was 🙏